remote/autostart

```diff
@@ -8,5 +8,5 @@ nitrogen --restore &
 if [ "$AUTOSTART_CAPELLA" = "1" ];
 then
     # Run capella in a loop:
-    ( while true; do /opt/capella/capella -data /workspace; sleep 1; done ) &
+    ( while true; do /opt/capella/capella -data /workspace > /var/log/capella.stdout.log 2> /var/log/capella.stderr.log; sleep 1; done ) &
```

Comment:
I would add a white space between '2' and '>'. And is the '&' needed?

Reply:
`2>` is the usual syntax to redirect `stderr`. The `&` symbol states that the process should run in the background and should continue without waiting for the endless while loop.